eng/Versions.props

```diff
@@ -17,8 +17,11 @@
     <!-- Enable to remove prerelease label. -->
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
-    <WorkloadVersionSuffix Condition="'$(DotNetFinalVersionKind)' != 'release' and '$(PreReleaseVersionIteration)' == '' and '$(PreReleaseVersionLabel)' != 'rtm'">-$(PreReleaseVersionLabel)</WorkloadVersionSuffix>
-    <WorkloadVersionSuffix Condition="'$(WorkloadVersionSuffix)' == '' and '$(DotNetFinalVersionKind)' != 'release' and '$(PreReleaseVersionLabel)' != 'rtm'">-$(PreReleaseVersionLabel).$(PreReleaseVersionIteration)</WorkloadVersionSuffix>
+    <_applyWorkloadVersionSuffix Condition="'$(DotNetFinalVersionKind)' != 'release'
+        and '$(PreReleaseVersionLabel)' != 'rtm'
+        and '$(PreReleaseVersionLabel)' != 'servicing'">true</_applyWorkloadVersionSuffix>
```

Comment:
Why do we need the `PreReleaseVersionLabel` checks?  Are those duplicating some policy or logic elsewhere?  If so can we comment about what this needs to be in sync with?